.github/renovate.json

```diff
@@ -3,23 +3,14 @@
     "github>rancher/renovate-config#release",
     "group:all"
   ],
-  "packageRules": [
-    {
-      "packageNames": [
-        "github.com/rancher/hull",
-        "github.com/rancher/lasso",
-        "github.com/rancher/wrangler/v2"
-      ],
-      "enabled": false,
-      "branches": [
-        "release/v3.0",
-        "release/v4.0"
-      ]
-    }
-  ],
   "baseBranches": [
     "release/v3.0",
     "release/v4.0",
     "release/v5.0"
+  ],
+  "ignoreDeps": [
+    "github.com/rancher/hull",
+    "github.com/rancher/lasso",
+    "github.com/rancher/wrangler/v2"
```

Comment:
Not sure if `github.com/rancher/wrangler` is in use in the older release branches, but if it is we should ignore that as well.

Also adding `github.com/rancher/wrangler/v3` ahead of time is not a bad idea here, since that is released or will release soon (i forget)

Reply:
That's a great point I overlooked and some very solid foresight.

Reply:
Wrangler 0-3 now included.